django/db/backends/base/features.py

```diff
@@ -323,6 +323,13 @@ class BaseDatabaseFeatures:
         'swedish_ci': None  # Swedish case-insensitive.
     }
 
+    # A set of dotted paths to tests in Django's test suite that are expected
+    # to fail on this database.
+    django_test_expected_failures = {}
```

Comment:
Oops, this should be `set()`. None of the subclasses implementations call `super()` which perhaps isn't necessary but I want the CockroachDB backend to call `super()` to get any skips/expected failures from the PostgreSQL backend  (which the CockroachDB inherits),

Reply:
I can create a PR to fix it

Reply:
Here is the PR https://github.com/django/django/pull/13761

Reply:
Good catch :dart: